openbas-front/src/admin/components/payloads/Payloads.tsx

```diff
@@ -101,13 +112,15 @@ const Payloads = () => {
   const dispatch = useAppDispatch();
 
   const [selectedPayload, setSelectedPayload] = useState<PayloadStore | null>(null);
-  const { documentsMap, collectorsMap } = useHelper((helper: DocumentHelper & CollectorHelper) => ({
+  const { documentsMap, collectorsMap, attackPatternsMap } = useHelper((helper: DocumentHelper & CollectorHelper & AttackPatternHelper) => ({
     documentsMap: helper.getDocumentsMap(),
     collectorsMap: helper.getCollectorsMap(),
+    attackPatternsMap: helper.getAttackPatternsMap(),
   }));
   useDataLoader(() => {
     dispatch(fetchDocuments());
     dispatch(fetchCollectors());
+    dispatch(fetchAttackPatterns());
```

Comment:
todo: remove this line
taxonomics are now retrieved at the login no need to fetch it later